packages/contracts-bedrock/test/universal/BenchmarkTest.t.sol

```diff
@@ -30,23 +30,51 @@ contract SetPrevBaseFee_Test is CommonTest {
 
 contract GasBenchMark_L1CrossDomainMessenger is CommonTest {
     function test_sendMessage_benchmark_0() external {
+        // Skip if the test is running in coverage.
+        if (vm.isContext(VmSafe.ForgeContext.Coverage)) {
+            vm.skip(true);
+        }
+
+        // Setup
         vm.pauseGasMetering();
+        uint256 startingGas = gasleft();
         setPrevBaseFee(vm, address(optimismPortal2), 1 gwei);
         // The amount of data typically sent during a bridge deposit.
         bytes memory data =
             hex"ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff";
         vm.resumeGasMetering();
+
+        // Test
         l1CrossDomainMessenger.sendMessage(bob, data, uint32(100));
+
+        // Assert
+        vm.pauseGasMetering();
+        assert(startingGas - gasleft() < 400_000);
```

Comment:
Also where does this magic 400k value come from? 
```suggestion
        assertLt(startingGas - gasleft(), 400_000);
```

Reply:
See https://github.com/ethereum-optimism/optimism/pull/14153#discussion_r1943539349